### DISCUSSION TITLE
De onde tirar criatividade??

### DISCUSSION THREAD

**Discussion by dtfialho** [frontendbr/forum#1214] [Discussão]
Fala pessoal, queria levantar uma discussão sobre uma coisa que é muito importante no nosso trabalho como desenvolvedores frontend, e que pode acontecer até mesmo com desenvolvedores mais experientes na hora de implementar algo novo.  Como vocês fazem na hora que a criatividade simplesmente vai embora? Quais as suas inspirações ou como fazem para ter ideias sobre o que fazer?

**Comment by MichelleTrigueiro** [comment]
Como ter ideia propriamente dito, pra mim é difícil falar, pois sou estudante ainda desse universo chamado desenvolvimento, mas como graduada em artes, sempre aconselho ter como livro de cabeceira o livro Gestalt do Objeto de João Gomes Filho

_Abstract: É consenso que a capacidade da informação visual é muito mais ampla do que aquelas transmitidas e/ou assimiladas pelos outros sentidos. A comunicação, hoje, é feita e difundida pelos mais avançados meios tecnológicos, meios estes que nos expõem - muitas vezes compulsoriamente - a múltiplos elementos visuais. Diante desta realidade, o aprendizado e a absorção de uma gramática da imagem tornam-se imprescindíveis à compreensão da forma de como se apresenta nossa cultura atual. Gestalt do objeto constitui uma importante contribuição para este entendimento._

**Comment by kvnol** [comment]
*A criatividade é natural*, vem quando estamos com a cabeça vazia. Por isso que temos as melhores ideias quando estamos próximos a dormir, tomando banho ou deitados no sofá. Porque damos tempo para a cabeça raciocinar os problemas.

Técnicas boas que eu uso e dá certo é: Pomodoro, as pausas ajudam MUITO;

E sites de inspiração, como o [Dribbble](https://dribbble.com/) e o [Designspiration](https://designspiration.net).

**Comment by diessica** [comment]
Eu acho que entender o que criatividade _significa_, e de onde ela vem, ajuda. Escrevi sobre isso: [Creativity](https://diessi.ca/blog/creativity/) (em inglês) – tem algumas referências de livros também, que você pode checar.

Gosto de acreditar que é algo que se pode exercitar, praticar como qualquer outra habilidade.  Inspiração para criar não vem do nada. Precisamos ter ideias para conectar para exercer a criatividade (e por isso, também, é importante estar cercado das coisas certas).

**Comment by lucasferreiralimax** [comment]
É sempre bom ter um senso de qualidade para essas horas, manter um olhar critico sobre oque é preciso reinventar ou não.

Eu costumo usar os sonhos para pensar melhor, aprender a dominar a imaginação é incrível.

Se não me vejo nada criativo busco inspiração na própria vida, nada que ela não ensine a criar novas formas e figuras ou perspectivas.

Dou uma volta de  bike no parque para ver a natureza além de fazer muito bem para o corpo ajuda a clarear novas ideias e circular os pensamentos.

**Comment by felipefialho** [comment]
Sempre tive a impressão de que a inspiração para o processo criativo só vem, especialmente quando não estamos tentando ser criativos, hehe
_____

> Não, não precisa se estar nem feliz nem aflito
Nem se refugiar em lugar mais bonito
Em busca da inspiração
Não, ela é uma luz que chega de repente
Com a rapidez de uma estrela cadente
Que acende a mente e o coração

Poder da Criação, Paulo César Pinheiro e João Nogueira

**Comment by leandrocunha** [comment]
@LFeh é o famoso ócio criativo, para que a gente exercite os nossos pensamentos precisamos "parar" de pensar em algo. Quando a gente fica pensando muito em precisar ser criativo não damos espaço para a nossa mente pensar no que precisa.

Quanto mais ficamos preocupados em ser criativos menos tempo temos de exercer nossa criatividade.

Meio doido mais é isso, na pratica, se quiser criar saia da frente do computador vai fazer outra coisa relaxante que o insight vai surgir.

**Comment by felipefialho** [comment]
@leandrocunha O proprio frontendbr surgiu assim 😄

**Comment by giovannism20** [comment]
https://www.youtube.com/watch?v=o7_tLOWPpyg
